library/core/qsl-base/src/main/java/org/quiltmc/qsl/api/event/ArrayEvent.java

```diff
@@ -0,0 +1,191 @@
+package org.quiltmc.qsl.api.event;
```

Comment:
don't forget to update this depending on what we decide the package structure is in the vote